INSTALL.rst

```diff
@@ -87,65 +89,80 @@ For example, on Debian GNU/Linux, you can install them as follows:
                         rabbitmq-server \
                         redis-server
 
-and do a system install for the Sass preprocessor by following `Sass web guide <http://sass-lang.com/install>`_ and running:
+Now, add the following lines in your "elasticsearch.yml" (for
+Debian GNU/Linux the full path is
+``/etc/elasticsearch/elasticsearch.yml``):
 
 .. code-block:: shell
 
-  sudo npm install -g sass node-sass clean-css uglify-js requirejs
-
+  # MY CONFIGS
 
-Installation
-^^^^^^^^^^^^
+  cluster.name: cap
+  discovery.zen.ping.multicast.enabled: false
+  http.port: 9200
+  http.publish_port: 9200
 
-Let's start by creating a new virtual environment that will hold our CAP
-v3.0 instance:
+Finally, do a system install (see below for how to do a local install
+instead) for the Sass preprocessor by following
+`Sass web guide <http://sass-lang.com/install>`_ and running:
 
 .. code-block:: shell
 
-   mkvirtualenv cap
+  sudo npm install -g node-sass clean-css uglify-js requirejs
 
-Start redis server in the background:
+Installation
+^^^^^^^^^^^^
+
+Let's start by cloning the repository:
 
 .. code-block:: shell
 
-   redis-server &
+   git clone https://github.com/cernanalysispreservation/analysis-preservation.cern.ch.git cap
+
+Environment Setup
+"""""""""""""""""
 
-Install CAP package:
+All else will be installed inside a python *virtualenv* for easy
+maintenance and encapsulation of the libraries required. To do so,
+create a new virtual environment to hold our CAP instance
+from inside the repository folder:
 
 .. code-block:: shell
 
-   cdvirtualenv
-   mkdir src && cd src
-   git clone https://github.com/cernanalysispreservation/analysis-preservation.cern.ch.git cap
    cd cap
-   pip install -r requirements.txt
+   mkvirtualenv cap_venv
 
-Add the following lines in your "elasticsearch.yml":
+Start redis server in the background:
 
 .. code-block:: shell
 
-  # MY CONFIGS
-
-  cluster.name: cap
-  discovery.zen.ping.multicast.enabled: false
-  http.port: 9200
-  http.publish_port: 9200
+   redis-server &
 
-Run npm to install any necessary JavaScript assets the Invenio modules
+Install the CAP package from inside your ``cap`` repository folder and
+run npm to install the necessary JavaScript assets the Invenio modules
 depend on:
 
 .. code-block:: shell
 
-   cd cap
+   pip install -r requirements.txt
    cap npm
    cdvirtualenv var/cap-instance/static
    npm install bower
    npm install
+
+Install alpaca:
+
+.. code-block:: shell
+
    cd node_modules/alpaca
    npm install
    npm start
```

Comment:
Yes.

With alpaca 1.5.17 in your ```package.json``` (or with current changes from Javi, has to be manually installed probably), the alpaca-needed libraries rest on the ```dist``` directory which is build automatically

but you need to have ```dist``` instead of ```build``` in your ```cap-settings.js``` like so:
https://github.com/cernanalysispreservation/analysis-preservation.cern.ch/commit/f1f7c292fce2e20743c69630ec8e5759f7ed6690#diff-270e1d08646842a9090903b0fe36608dR32


Here is what I have:
```
(cap)➜  alpaca cdvirtualenv var/cap-instance/static/node_modules/alpaca
(cap)➜  alpaca l
total 48
drwxr-xr-x   11 pamfilos  staff   374B Aug 30 17:27 .
drwxr-xr-x  270 pamfilos  staff   9.0K Aug 30 17:28 ..
-rw-r--r--    1 pamfilos  staff    11K Mar 15 14:21 README.md
-rw-r--r--    1 pamfilos  staff   2.2K Mar 15 14:21 bower.json
drwxr-xr-x   34 pamfilos  staff   1.1K Aug 30 17:27 bower_components
drwxr-xr-x    4 pamfilos  staff   136B Aug 30 17:27 dist
-rw-r--r--    1 pamfilos  staff   688B Mar 15 14:21 license.txt
-rw-r--r--    1 pamfilos  staff   3.2K Aug 30 17:27 package.json
drwxr-xr-x    5 pamfilos  staff   170B Aug 30 17:27 src
drwxr-xr-x   10 pamfilos  staff   340B Aug 30 17:27 tests
drwxr-xr-x    3 pamfilos  staff   102B Mar 15 14:21 thirdparty
```